packages/cli/src/Server.ts

```diff
@@ -2600,6 +2612,23 @@ class App {
 			}),
 		);
 
+		// ----------------------------------------
+		// Binary data
+		// ----------------------------------------
+
+		// Returns binary buffer
+		this.app.get(
+			`/${this.restEndpoint}/data/:path`,
+			ResponseHelper.send(async (req: express.Request, res: express.Response): Promise<string> => {
+				const dataPath = req.params.path;
+				return BinaryDataManager.getInstance()
```

Comment:
We should add a catch block to prevent users from seeing too much info such as filesystem structure / files deployment.